src/EventMetadata.php

```diff
@@ -10,7 +10,19 @@
 class EventMetadata extends RequestMetadata
 {
     /**
-     * @var int Time the event occurred.
+     * @var string Unique ID of tenant that is performing the operation.
+     */
+    private $tenantId;
+    /**
+     * @var IclFields Metadata about the request for the Tenant Security Proxy to log.
+     */
+    private $iclFields;
+    /**
+     * @var string[] Optional additional information for the Tenant Security Proxy to log.
+     */
+    private $customFields;
+    /**
+     * @var int Linux epoch millis of when the event occurred.
```

Comment:
These are already in RequestMetadata - why add them again?

Reply:
We have 2 types of metadata - RequestMetadata and EventMetadata. This one mistakenly was not declaring its variables (but was setting them in its constructor). Starting in 8.2, dynamically setting variables like that is deprecated.